api/events/msam-events-release.json

```diff
@@ -35,6 +35,7 @@
                                 "Effect": "Allow",
                                 "Action": [
                                     "dynamodb:PutItem",
+                                    "medialive:Describe*",
```

Comment:
This file got checked in because of the addition of this line.

Reply:
This file is dynamically generated based on MSAMEventCollector.yml so no need to make changes here.

Reply:
We only check in this file (msam-events-release.json) when we're making an official release. So I'd remove this from the PR.

Reply:
@morjoan thanks for the insight. will remove from PR